include/veriblock/blockchain/blocktree.hpp

```diff
@@ -188,20 +191,20 @@ struct BlockTree : public BaseBlockTree<Block> {
                            bool shouldContextuallyCheck,
                            index_t** ret) {
     if (!checkBlock(*block, state, *param_)) {
-      return state.Invalid("check-block");
+      return state.Invalid(block_t::name() + "check-block");
     }
 
     // we must know previous block
     auto* prev = base::getBlockIndex(block->previousBlock);
     if (prev == nullptr) {
       return state.Invalid(
-          "bad-prev-block",
+          block_t::name() + "bad-prev-block",
```

Comment:
I think it should be like block_t::name() + "-bad-prev-block" otherwise it would become unreadable.
The same applies for other snippets below.